src/pages/routes.test.ts

```diff
@@ -0,0 +1,41 @@
+import fs from 'fs'
+import { parseStringPromise } from 'xml2js'
+
+import { routes } from './RouteDefinitions'
+
+describe('Routes', () => {
+  it('sitemap URLs should exist as Router paths', async () => {
+    const pathNames: string[] = routes.map((routeDef) => routeDef.path)
+    await new Promise<boolean>((resolve, reject) => {
+      try {
+        fs.readFile('./public/sitemap.xml', 'utf8', async (err, data) => {
+          if (err) {
+            reject(err)
+          }
```

Comment:
nit: you could simplify this, you have a try-catch around it
```suggestion
          if (err) throw err
```